draft-dcook-ppm-dap-interop-test-design.md

```diff
@@ -277,6 +280,25 @@ algorithms in section 6 of [DAP], for broad compatibility.
 {: title="Response JSON object structure"}
 
 
+### `/internal/test/add_authentication_token` {#aggregator-add-auth-token}
+
+Associate an authentication token with a task, for the purposes of making
+authenticated requests to other protocol participants or validating
+authenticated requests from other protocol participants. Authenticated requests
+MUST include the token in an HTTP header named `DAP-Auth-Token`.
+
+|Key|Value|
+|`task_id`|A base64url-encoded DAP `TaskId`.|
+|`role`|Either `"leader"` or `"collector"`, indicating which type of authentication token this is.|
+|`token`|The authentication token, as a string. This string must be safe for use as an HTTP header value.|
```

Comment:
Just for my own edification: what's an example of a string that's not a safe HTTP header value?

Reply:
Low ASCII bytes or non-ASCII bytes are not allowed, so it has to be all 0x20-0x7F.